### ISSUE TITLE
mayaUsdRegistry error with 0.11.0 and Arnold 4.2.3 

### ISSUE THREAD

**Issue by jcoelho88** [Autodesk/maya-usd#1625]
Hello,

I'm wondering if anyone else is having the same problem I'm having.

Currently I'm using Maya 2022.1, maya-usd-0.11.0 and arnold-4.2.3 and have encountered an issue when trying to do a USD Export. I expect to see Arnold as an option in the material dropdown in the export dialog window but instead this error appears and no option for Arnold appears. 
![image](https://user-images.githubusercontent.com/88895104/129376295-38356082-45dc-4324-8820-3ffa743dcb61.png)

Here is the message:
```
// Error: Failed to load plugin 'mayaUsdRegistry': The specified procedure could not be found.
 in 'c:/Program Files/Autodesk/Arnold/maya2022/usd/mayaUsdRegistry/2102/mayaUsdRegistry.dll' // 
```
 This does not occur when I install maya-usd-0.10.0

I have uninstalled all previous versions and just kept 0.11.0 but I still have the issue.

Any help on this would be appreciated.

Cheers,

Jason Coelho

_Originally posted by @jcoelho88 in https://github.com/Autodesk/maya-usd/discussions/1623_

**Comment by sebastienblor** [comment]
Hi Jason, 
indeed MtoA 4.2.3  was released before 0.11 was out, and its usd modules were built for maya-usd 0.10. 
Since these versions aren't binary compatible, it's not loading anymore.
You will need a newer version of MtoA, that is yet to be released.
Cheers

**Comment by jcoelho88** [comment]
Hi Sebastien,

Thanks for the response!

I was wondering if that may have been the case, thanks for the info.

I'll close this issue and keep an eye out for the next MtoA release.

Cheers,

Jason Coelho
